contracts/oracles/IporRiskManagementOracle.sol

```diff
@@ -0,0 +1,261 @@
+    ) internal {
+        IporRiskManagementOracleStorageTypes.RiskIndicatorsStorage memory indicators = _indicators[asset];
+
+        require(indicators.lastUpdateTimestamp > 0, IporRiskManagementOracleErrors.ASSET_NOT_SUPPORTED);
+
+        _indicators[asset] = IporRiskManagementOracleStorageTypes.RiskIndicatorsStorage(
+            maxNotionalPayFixed.toUint64(),
+            maxNotionalReceiveFixed.toUint64(),
+            maxUtilizationRatePayFixed.toUint16(),
+            maxUtilizationRateReceiveFixed.toUint16(),
+            maxUtilizationRate.toUint16(),
+            block.timestamp.toUint32()
+        );
+
+        emit RiskIndicatorsUpdated(
+            asset,
+            maxNotionalPayFixed,
+            maxNotionalReceiveFixed,
+            maxUtilizationRatePayFixed,
+            maxUtilizationRateReceiveFixed,
+            maxUtilizationRate
+        );
+    }
+
+    function addAsset(
+        address asset,
+        uint256 maxNotionalPayFixed,
+        uint256 maxNotionalReceiveFixed,
+        uint256 maxUtilizationRatePayFixed,
+        uint256 maxUtilizationRateReceiveFixed,
+        uint256 maxUtilizationRate
+    ) external override onlyOwner {
+        require(asset != address(0), IporErrors.WRONG_ADDRESS);
+        require(
+            _indicators[asset].lastUpdateTimestamp == 0,
+            IporRiskManagementOracleErrors.CANNOT_ADD_ASSET_ASSET_ALREADY_EXISTS
+        );
+
+        _indicators[asset] = IporRiskManagementOracleStorageTypes.RiskIndicatorsStorage(
+            maxNotionalPayFixed.toUint64(),
+            maxNotionalReceiveFixed.toUint64(),
+            maxUtilizationRatePayFixed.toUint16(),
+            maxUtilizationRateReceiveFixed.toUint16(),
+            maxUtilizationRate.toUint16(),
+            block.timestamp.toUint32()
+        );
+
+        emit IporRiskManagementOracleAssetAdded(asset);
+    }
+
+    function removeAsset(address asset) external override onlyOwner {
+        require(asset != address(0), IporErrors.WRONG_ADDRESS);
+        require(_indicators[asset].lastUpdateTimestamp > 0, IporRiskManagementOracleErrors.ASSET_NOT_SUPPORTED);
+
+        delete _indicators[asset];
+        emit IporRiskManagementOracleAssetRemoved(asset);
+    }
+
+    function isAssetSupported(address asset) external view override returns (bool) {
+        return _indicators[asset].lastUpdateTimestamp > 0;
+    }
+
+    function pause() external override onlyOwner {
+        _pause();
+    }
+
+    function unpause() external override onlyOwner {
+        _unpause();
+    }
+
+    function addUpdater(address updater) external override onlyOwner {
+        require(updater != address(0), IporErrors.WRONG_ADDRESS);
+
+        _updaters[updater] = 1;
+        emit IporRiskManagementOracleUpdaterAdded(updater);
+    }
+
+    function removeUpdater(address updater) external override onlyOwner {
+        require(updater != address(0), IporErrors.WRONG_ADDRESS);
+
+        _updaters[updater] = 0;
```

Comment:
do you know the difference between deleting and assigning zero? is it cheaper or not?